---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

<!-- {% if site.author.googlescholar %}
  <div class="wordwrap">You can also find my articles on <a href="{{site.author.googlescholar}}">my Google Scholar profile</a>.</div>
{% endif %} -->

{% include base_path %}

<div class="publications">

    <div class="publication-item">
    <p>
      <!-- <strong>Accurate reconstruction of circular RNAs from complex rolling circular long reads with CircPlex</strong> -->
      Accurate reconstruction of circular RNAs from complex rolling circular long reads with CircPlex
      [<a href="https://doi.org/10.1101/2025.11.21.689841" target="_blank">preprint</a>]
      [<a href="https://github.com/Shao-Group/CircPlex" target="_blank">code</a>]<br>
      <strong>Tasfia Zahin</strong>, Irtesam Mahmud Khan, Mingfu Shao<br>
      <em>bioarxiv</em>
    </p>
  </div>

  <div class="publication-item">
    <p>
      <!-- <strong>Accurate Detection of Tandem Repeats from Error-Prone Sequences with EquiRep</strong> -->
      Accurate Detection of Tandem Repeats from Error-Prone Sequences with EquiRep
      [<a href="https://doi.org/10.1101/gr.280750.125" target="_blank">paper</a>]
      [<a href="https://github.com/Shao-Group/EquiRep" target="_blank">code</a>]<br>
      Zhezheng Song<sup>*</sup>, <strong>Tasfia Zahin</strong><sup>*</sup>, Xiang Li, Mingfu Shao<br>
      <em>Genome Research 2025</em>
    </p>
  </div>

  <div class="publication-item">
    <p>
      <!-- <strong>Accurate Detection of Tandem Repeats from Error-Prone Sequences with EquiRep</strong> -->
      Accurate Detection of Tandem Repeats from Error-Prone Sequences with EquiRep
      [<a href="https://link.springer.com/chapter/10.1007/978-3-031-90252-9_46" target="_blank">paper</a>]
      [<a href="https://doi.org/10.1101/2024.11.05.621953" target="_blank">preprint</a>]
      [<a href="/files/RECOMB25_EquiRep.pdf" target="_blank">slide</a>]<br>
      Zhezheng Song<sup>*</sup>, <strong>Tasfia Zahin</strong><sup>*</sup>, Xiang Li, Mingfu Shao<br>
      <em>Proceedings of 29th International Conference on Research in Computational Molecular Biology (RECOMB 2025)</em>
    </p>
  </div>

  <div class="publication-item">
    <p>
      <!-- <strong>Accurate Assembly of Circular RNAs with TERRACE</strong> -->
      Accurate Assembly of Circular RNAs with TERRACE
      [<a href="https://genome.cshlp.org/content/34/9/1365" target="_blank">paper</a>]
      [<a href="https://github.com/Shao-Group/TERRACE" target="_blank">code</a>]<br>
      <strong>Tasfia Zahin</strong>, Qian Shi<sup>*</sup>, Carl Zang<sup>*</sup>, Mingfu Shao<br>
      <em>Genome Research 2024</em>
    </p>
  </div>

  <div class="publication-item">
    <p>
      <!-- <strong>Accurate Assembly of Circular RNAs with TERRACE</strong> -->
      Accurate Assembly of Circular RNAs with TERRACE
      [<a href="https://link.springer.com/chapter/10.1007/978-1-0716-3989-4_49" target="_blank">paper</a>]
      [<a href="https://doi.org/10.1101/2024.02.09.579380" target="_blank">preprint</a>]
      [<a href="/files/RECOMB24-TERRACE.pdf" target="_blank">slide</a>]<br>
      <strong>Tasfia Zahin</strong>, Qian Shi<sup>*</sup>, Carl Zang<sup>*</sup>, Mingfu Shao<br>
      <em>Proceedings of 28th International Conference on Research in Computational Molecular Biology (RECOMB 2024)</em>
    </p>
  </div>

  <div class="publication-item">
    <p>
      <!-- <strong>Amaranth: enhanced single-cell transcript assembly via discriminative modeling of UMI reads and internal reads</strong> -->
      Amaranth: enhanced single-cell transcript assembly via discriminative modeling of UMI reads and internal reads
      [<a href="https://doi.org/10.1101/2025.11.24.690228" target="_blank">preprint</a>]
      [<a href="https://github.com/Shao-Group/amaranth" target="_blank">code</a>]<br>
      Xiaofei Carl Zang, <strong>Tasfia Zahin</strong>, Irtesam Mahmud Khan, Qian Shi, Yi Xing, Mingfu Shao<br>
      <em>bioarxiv</em>
    </p>
  </div>

  <div class="publication-item">
    <p>
      <!-- <strong>Boosting transcript assembly via delineating transcript start and end sites</strong> -->
      Boosting transcript assembly via delineating transcript start and end sites
      [<a href="https://doi.org/10.1101/2025.10.13.682211" target="_blank">preprint</a>]
      [<a href="https://github.com/Shao-Group/Telos" target="_blank">code</a>]<br>
      Irtesam Mahmud Khan, Xiaofei Carl Zang, Ange Teng, <strong>Tasfia Zahin</strong>, Mingfu Shao<br>
      <em>bioarxiv</em>
    </p>
  </div>

  <div class="publication-item">
    <p>
      <!-- <strong>An alignment-free method for phylogeny estimation using maximum likelihood</strong> -->
      An alignment-free method for phylogeny estimation using maximum likelihood
      [<a href="https://doi.org/10.1186/s12859-025-06080-w" target="_blank">paper</a>]
      [<a href="https://github.com/hasin-abrar/Peafowl-repo" target="_blank">code</a>]<br>
      <strong>Tasfia Zahin</strong><sup>*</sup>, Md Hasin Abrar<sup>*</sup>, Mizanur Rahman Jewel, Tahrina Tasnim, Md. Shamsuzzoha Bayzid, Atif Rahman<br>
      BMC Bioinformatics 2025
    </p>
  </div>

</div>

<!-- Add footnote here -->
<hr>
<p style="font-size: 12px;"><strong>*</strong> Equal contribution</p>
